pkg/store/connection/connection_recorder.go

```diff
@@ -131,6 +131,44 @@ func (c *Recorder) SaveNamespaceThreadID(threadID, namespace, connectionID strin
 	return c.transientStore.Put(getNamespaceKeyPrefix(prefix)(key), []byte(connectionID))
 }
 
+// RemoveConnection removes connection record from the store for given id
+func (c *Recorder) RemoveConnection(connectionID string) error {
+	record, err := c.GetConnectionRecord(connectionID)
+	if err != nil {
+		return err
```

Comment:
Better to wrap an error with some helpful message here.

Reply:
Did it.